<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiGestionActeur</class>
 <widget class="QWidget" name="GuiGestionActeur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion des acteurs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitreGestionActeur">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Liste des acteurs :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblFiltrer">
       <property name="text">
        <string>Filtrer par r√¥le :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFiltreRole"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnToutAfficher">
       <property name="text">
        <string>Tout Afficher</string>
       </property>
       <property name="icon">
        <iconset resource="../../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/toutAfficher.png</normaloff>:/SeboApp/Resources/toutAfficher.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tvActeur">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAjouter">
         <property name="text">
          <string>Ajouter</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/add2.png</normaloff>:/SeboApp/Resources/add2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModifier">
         <property name="text">
          <string>Modifier</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/modifier2.png</normaloff>:/SeboApp/Resources/modifier2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSupprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/delete2.png</normaloff>:/SeboApp/Resources/delete2.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../SeboApp.qrc"/>
 </resources>
 <connections/>
</ui>
